alpinelib/aws/msk.py

```diff
@@ -90,18 +90,16 @@ def _new_producer(producer_name: str, cluster_name: str, **kwargs) -> KafkaProdu
             logger.warning("Did not create producer as no bootstrap servers could be found")
             pass
     except Exception as e:
-        logger.exception("Failed to connect to cluster {} with broker string {}.".format(cluster_name,bootstrap_servers))
+        logger.exception(
+            "Failed to connect to cluster {} with broker string {}.".format(cluster_name, bootstrap_servers))
         raise e
 
 
-def _get_broker_tls_string(cluster_name: str):
+def _get_broker_tls_string(bucket_name: str, cluster_name: str):
     try:
-        return read_object('de-kafka-connections', cluster_name+'.txt')
+        return read_object(bucket_name, cluster_name + '.txt')
     except Exception as e:
-        message = 'Could not find broker connection file named {} in bucket de-kafka-connections'.format(cluster_name+'.txt')
+        message = 'Could not find broker connection file named {} in bucket de-kafka-connections'.format(
```

Comment:
Should probably remove the reference in the log message as well

Reply:
Good catch

Reply:
Done